src/driver/aurora-data-api/AuroraDataApiQueryRunner.ts

```diff
@@ -1340,7 +1344,7 @@ export class AuroraDataApiQueryRunner extends BaseQueryRunner implements QueryRu
                     table: table,
                     name: constraint["INDEX_NAME"],
                     columnNames: indices.map(i => i["COLUMN_NAME"]),
-                    isUnique: constraint["NON_UNIQUE"] === "0",
+                    isUnique: constraint["NON_UNIQUE"] == "0", // tslint:disable-line:triple-equals - trying to find truthy value
```

Comment:
imagine somebody reading this code. Can you please explicitly specify what logic should be in there (as I understood we should check for === "0" && === 0? If so, please change it

Reply:
@pleerock updated the code to be more explicit.

Pulled in latest from master and rebased.

I debated pulling that logic out into a method on the QueryRunner, but didn't want to overstep and make assumptions.
something like:
`private isUniqueIndex(constraint): boolean`

If you'd rather go that route, I can update. Otherwise let me know if there's anything else you'd like changed.